modules/utils/src/merkle.ts

```diff
@@ -15,7 +16,9 @@ export const generateMerkleTreeData = (transfers: CoreTransferState[]): { root:
   });
 
   // Generate tree
-  const tree = new MerkleTree(leaves, keccak256, { sortPairs: true });
+  const tree = new merkle.Tree();
+  tree.insert_hex_js(leaves);
```

Comment:
`insert_hex_js` expects a string from `encodeCoreTransferState`, not `hashCoreTransferState`. It decodes the `transferId` from there, and does it's own hashing.